### DISCUSSION TITLE
Get reference key for a paper

### DISCUSSION THREAD

**Discussion by apraga** [papis/papis#728] [Q&A] [answered]
Hi,

I would like to get only the reference/citation key of a paper (use case: get the reference a paper in a markdown file).
I cannot figure out how to do it from the documentation unfortunately.
`papis list` does not make it available. I've even tried to get it from bibtex but it does not print anything : 

    papis bibtex read biblio.bib ref -o test

Thanks,

**Comment by alexfikl** [comment, marked as answer]
Unless I'm misunderstanding, you could do this with the `papis list` command, e.g.
```
papis list --format '{doc[ref]}' <QUERY>
```
should list references for all documents that have them.

**Comment by apraga** [comment]
Hi alexfiki,
Thanks, that's exactly what I'm looking for but could not find out from the documentation. That's why I'm posting there in case it helps other people.
